app/scripts/lib/ppom/ppom-util.test.ts

```diff
@@ -60,8 +60,49 @@ const TRANSACTION_PARAMS_MOCK_1: TransactionParams = {
 };
 
 const SIGN_TYPED_DATA_PARAMS_MOCK_1 = '0x123';
-const SIGN_TYPED_DATA_PARAMS_MOCK_2 =
-  '{"primaryType":"Permit","domain":{},"types":{}}';
+const SIGN_TYPED_DATA_PARAMS_MOCK_2 = {
```

Comment:
Minor, as we have dedicated tests for the module, is this overlap needed?

Reply:
Removed overlapping test https://github.com/MetaMask/metamask-extension/pull/29698/commits/cdb8f6de50e619e0424689b4855cc0cfed60c6b0